tools/accuracy_checker/README.md

```diff
@@ -88,21 +88,23 @@ All relative paths in config files will be prefixed with values specified in com
 You may refer to `-h, --help` to full list of command line options. Some optional arguments are:
 
 - `-r, --root` prefix for all relative paths.
```

Comment:
hmm... we do not have this command line parameter at all :)

Reply:
Parameter -r has been removed from description